realm/realm-library/src/androidTest/java/io/realm/RxJavaTests.java

```diff
@@ -99,6 +99,8 @@ private void disposeSuccessfulTest(BaseRealm testRealm) {
                 @Override
                 public void run() {
                     // Wait for Subscription to dispose of external resources
+                    RealmLog.error("Global: " + Realm.getGlobalInstanceCount(testRealm.getConfiguration()));
+                    RealmLog.error("Local: " + Realm.getGlobalInstanceCount(testRealm.getConfiguration()));
```

Comment:
Should be local

Reply:
Yeah, I already fixed it. Trying to locate a bug that only manifest on CI. 20 minute roundtrip times when debugging is fun...or something 😆